duo_client/client.py

```diff
@@ -221,12 +221,24 @@ def api_call(self, method, path, params):
         * params: dict mapping from parameter name to stringified value,
             or a dict to be converted to json.
         """
+        params_go_in_body = method in ('POST', 'PUT', 'PATCH')
         if self.sig_version in (1, 2):
             params = normalize_params(params)
-        elif self.sig_version in (3, 4):
+            # v1 and v2 canonicalization don't distinguish between
+            # params and body. There's no separate body input.
+            body = None
+        elif self.sig_version == 3:
             # Raises if params are not a dict that can be converted
             # to json.
             params = self.canon_json(params)
+            body = params
```

Comment:
This is a bit hacky, but since sig v3 is deprecated, we can remove this in a later PR.

Reply:
v3 is so deprecated we removed it from this codebase. I vote against adding it back. :)

Reply:
Yeah, I thought we had already removed it here. We did remove it from other code bases. We should get on that ASAP.

Reply:
I've got a diff in progress to do so. All red :)